ecs-agent/acs/session/task_manifest_responder.go

```diff
@@ -67,7 +67,7 @@ func NewTaskManifestResponder(taskComparer TaskComparer, snAccessor SequenceNumb
 		messageIDAccessor: messageIDAccessor,
 		metricsFactory:    metricsFactory,
 	}
-	r.respond = ResponseToACSSender(r.Name(), responseSender)
+	r.respond = responseToACSSender(r.Name(), responseSender)
```

Comment:
why is this uncapitalized? won't that make this method private?

Reply:
See PR description:
> since this function is no longer needed to be used outside of ecs-agent/acs/session/ and thus should be private

Reply:
ah I see, that should be ok.